### PR TITLE
feat: added login and register page

### PR DESCRIPTION
**Pull request by aryan-13**
-added login page
-added register page
-updated links to buttons

Preview: https://deploy-preview-3--lunarstore.netlify.app/login.html

### CODE REVIEW

**Review by amarphule — COMMENTED** [review]
looks good to me, One suggestion I think Just show `Login` heading on login page and `Register` heading on registration page.

File: css/layout/login.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    -webkit-box-shadow: 4px 4px 32px 0px rgba(194, 193, 193, 0.75);
+    -moz-box-shadow: 4px 4px 32px 0px rgba(194, 193, 193, 0.75);
+    font-size: 2rem;
+    row-gap: 2rem;
+}
+
+.login-input {
+    display: flex;
+    flex-direction: column;
+
```
Comment (ZuberDunge) [inline css/layout/login.css]: you can remove extra spacing

**Review by ZuberDunge — APPROVED** [review]
Code looks good, just avoid extra spacing from next time, rest everything looks good.

**Comment by hinanshisuthar** [comment]
demo

File: css/layout/login.css (hunk 1 above)
Comment (aryan-13) [reply css/layout/login.css]: Fixed the spacing and added comments

**Pull request merged**